java-lib/src/main/java/com/wavefront/ingester/HistogramDecoder.java

```diff
@@ -16,7 +16,7 @@
  */
 public class HistogramDecoder implements Decoder<String> {
   private static final Logger logger = Logger.getLogger(HistogramDecoder.class.getCanonicalName());
-  private static final IngesterFormatter FORMAT = IngesterFormatter.newBuilder()
+  private static final IngesterFormatter FORMAT = (IngesterFormatter) IngesterFormatter.newBuilder()
```

Comment:
Please use generics for this if possible (make AbstractIngesterFormatter take a type parameter)